app/controllers/participants/validations_controller.rb

```diff
@@ -0,0 +1,237 @@
+# frozen_string_literal: true
+
+module Participants
+  class ValidationsController < BaseController
```

Comment:
More out of curiosity than a strong opinion here, did you think about using the multiform that we have in `app/lib` directory?

Reply:
Yes, I left a comment in the PR description. I started out wanting to use it, but I couldn't get it to work and needed to get this done quickly. I also couldn't afford to find out that it wouldn't work with this logic for some reason. Unfortunately at the time Stan wasn't around to ask.  Thought we could refactor later if necessary, however I do think the multiform needs documentation as it isn't "obvious" (to me at least and I've written quite a few of those kind of things over the years).